executor.go

```diff
@@ -219,6 +221,42 @@ func (e *executor) validateCallArgs(c *pql.Call) error {
 	return nil
 }
 
+func (e *executor) executeQueryCall(ctx context.Context, index string, c *pql.Call, shards []uint64, opt *execOptions) (interface{}, error) {
+	optCopy := &execOptions{}
+	*optCopy = *opt
```

Comment:
why is the copy necessary? we can't use opt all the way through?

Reply:
oh, nevermind, I see why. the opts are reused if there are multiple calls in the query.